python-shell/src/test/test_gaffer_operations.py

```diff
@@ -5475,10 +5474,6 @@ def test_get_walks_correct_op(self):
         self.assertEqual(g.GetWalks(
             operations=[g.GetElements()]), expected_output_json)
 
-    def test_get_walks_incorrect_op(self):
```

Comment:
The incorrect get_walks tests have been removed as the error will now come directly from the Gaffer api (500) rather than throwing first in Python.